tests/data/corrections/DECam-ct4m20121211t220632.yaml

```diff
@@ -0,0 +1 @@
+DETECTOR: NEW-ID
```

Comment:
I thought we had agreed (maybe just by convention) that detector ids were integers.

Reply:
This value is something for me to use in a test of header fixing. It's not being used in metadata translation at all -- I am checking that when a try to fix a header that purports to have this OBSID, that the detector header is replaced with that value.

Reply:
Also, in DECam the DETECTOR header refers to the detector serial. It's not the detector name or number.